_posts/2024-05-31-friends-dont-let-friends-package-alone-pyconus-24.md

```diff
@@ -0,0 +1,282 @@
+practicing perfect lettering, using markers, fancy pens and colored pencils  to bring creative
+landscape visions to life.
+
+Now, as a Pythonista, coder, data scientist, and tech
+geek, I’m very aware of how much more others know than I do about lots of
+different topics—including packaging.
+
+I wasn’t sure what it would be like to be on that big pyCon US presentation stage, in that big room full of chairs. Would they be empty? Luckily, I was able to focus on my friends sitting in the front rows
+
+Carol, Hugo, Jeremiah, and Inessa - they were all there.
+
+Carol told me to lean into the teacher inside of me. I know that teacher best. My body calmed as I remembered why I was there. To help the community move forward.
+
+And similar to how we created our packaging guide - my friends got me through it.
+
+## The core messages in my talk
+
+In my talk, I shared my experiences as a maintainer of [Stravalib](https://github.com/stravalib/stravalib). How
+challenging it was to figure out the best packaging infrastructure for our team.
+However, that experience of rebuilding stravalib also empowered me to better understand the pain points
+in the ecosystem from a "beginner and real world application lens." From a lens of "I just need to create a
+package and publish / release it to PyPI regularly."
+
+From a lens of - I am technical but I need to learn new-to-me packaging tools
+and workflows.
+
+<figure>
+  <img src="{{ site.baseurl }}/images/blog/2024/june/pyopensci-maintainer-python-packaging-frustration.png" alt="Image of a stressed out looing stick figure typing at a computer. On the left is says Maintainers Perspective." style="width: 80%;" />
+  <figcaption><i class="fa-solid fa-heart-pulse"></i> Even as technical maintainers, the folks on the stravalib team weren't packaging experts. Why would they be? They are focused on writing good usable code (and tests). <i class="fa-solid fa-heart-pulse"></i>.</figcaption>
+</figure>
+
+## Fundamental first - How to make a technical ecosystem beginner accessible
+
+How do we collectively support and empower people who don't want to become
+packaging gurus to be successful?
+
+My approach to helping beginners tackle hard technical problems has always been
+about breaking things down into their simplest forms first.
+
+I thrive on those “ah ha” moments when a tough concept clicks.
+
+And someone gets it.
+
+Sometimes, that someone is me — before I can teach the topic to others!
+
+## So what about packaging with friends?
+
+The essence of my Python packaging talk was the work that I have lead alongside the pyOpenSci community to demystify and help _people_ navigate the complex Python packaging ecosystem.
+
+This work was really about people helping people tackle challenging things.
+
+Python packaging matters because scientists need open software to create open and reproducible workflows. To build upon each other's work. To tackle the world's hardest challenges more efficiently and effectively.
```

Comment:
```suggestion
Python packaging matters because scientists need open software to create open and reproducible workflows to build upon each other's work, and to tackle the world's hardest challenges more efficiently and effectively.
```